packages/renderer/src/lib/table/Table.svelte

```diff
@@ -189,7 +189,8 @@ function setGridColumns() {
             {#if column.info.renderer}
               <svelte:component
                 this="{column.info.renderer}"
-                object="{column.info.renderMapping?.(object) ?? object}" />
+                object="{column.info.renderMapping?.(object) ?? object}"
+                on:update />
```

Comment:
I think this change should not be part of this PR that is focusing on pods 

(also it is missing a test case)

Reply:
Separated out into #5345 (with test).